demo/build.gradle

```diff
@@ -6,15 +6,20 @@ android {
 
     defaultConfig {
         applicationId "com.braintreepayments.browserswitch.demo"
-        minSdkVersion 15
+        minSdkVersion 21
```

Comment:
This change is actually part of PR #24 in branch `set-minSdkVersion-to-21`, but it appears in this PR as well, because I can't change the base branch of this PR to `set-minSdkVersion-to-21` which is only in my fork.